core/types/bodyv0.go

```diff
@@ -0,0 +1,84 @@
+package types
+
+import (
+	"io"
+
+	"github.com/ethereum/go-ethereum/rlp"
+
+	"github.com/harmony-one/harmony/block"
+	"github.com/harmony-one/harmony/internal/utils"
+)
+
+// BodyV0 is the V0 block body
+type BodyV0 struct {
+	f bodyFieldsV0
+}
+
+type bodyFieldsV0 struct {
+	Transactions []*Transaction
+	Uncles       []*block.Header
+}
+
+// Transactions returns the list of transactions.
+//
+// The returned list is a deep copy; the caller may do anything with it without
+// affecting the original.
+func (b *BodyV0) Transactions() (txs []*Transaction) {
+	for _, tx := range b.f.Transactions {
+		txs = append(txs, tx.Copy())
```

Comment:
Is this efficient? If it's called a lot. it will probably slow things down.

Reply:
Should be pretty fast, but nevertheless good call.  I'll add a few methods to help single-transaction use cases.